tests/Avalonia.Direct2D1.RenderTests/FontManagerImplTests.cs

```diff
@@ -1,17 +1,13 @@
-﻿using System;
-using System.Reflection;
-using Avalonia.Direct2D1.Media;
+﻿using Avalonia.Direct2D1.Media;
 using Avalonia.Media;
 using Avalonia.Platform;
-using Avalonia.UnitTests;
+using Avalonia.Shared.PlatformSupport;
 using Xunit;
 
-namespace Avalonia.Direct2D1.UnitTests.Media
+namespace Avalonia.Direct2D1.RenderTests
```

Comment:
Why were tests tests moved from UnitTests to RenderTests? As far as I can see they're actually unit tests, not render tests (they're not rendering to a bitmap etc).

Reply:
`AssetLoader` is also defined in `Avalonia.UnitTests` but `Avalonia.Skia.UnitTests` requires this reference. So I had to move the tests to `RenderTests`. First I tried to get around this by explicitly activateing the right type by using `Activator.CreateInstance`. but it caused issues at other places.

Reply:
> AssetLoader is also defined in Avalonia.UnitTests but Avalonia.Skia.UnitTests requires this reference. 

Sorry, I don't understand - requires which reference?

Reply:
`Avalonia.Skia.UnitTests' has a project reference to `Avalonia.UnitTests`

`AssetLoader` is defined in `Avalonia.Shared.PlatformSupport` and `Avalonia.UnitTests`